From 1779096c91b85b47669feecb7f5ff60b7c9b30d0 Mon Sep 17 00:00:00 2001
From: Pablo Rodriguez <pablo.rodriguez-quesada@windriver.com>
Date: Thu, 16 Apr 2020 19:28:41 -0600
Subject: [PATCH] zynqmp-zcu102-rev1.0: Fix mmc0 tuning failed clock error

This is a known issue in the 2019.2 ZCU106 BSP. It is happenning
in ZCU102 aswell. See: https://www.xilinx.com/support/answers/73079.html
The error is as follows:
mmc0: SDHCI controller on ff170000.mmc [ff170000.mmc] using ADMA 64-bit
mmc0: new ultra high speed DDR50 SDHC card at address aaaa
mmc0: Tuning failed, falling back to fixed sampling clock

It is fixed by disabling the 1.8 voltage on mmc0@ff170000 in DTB.

Signed-off-by: Pablo Rodriguez <pablo.rodriguez-quesada@windriver.com>
---
 arch/arm64/boot/dts/xilinx/zynqmp-zcu102-rev1.0.dts | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/arch/arm64/boot/dts/xilinx/zynqmp-zcu102-rev1.0.dts b/arch/arm64/boot/dts/xilinx/zynqmp-zcu102-rev1.0.dts
index f44a4b5..a77ee1b 100644
--- a/arch/arm64/boot/dts/xilinx/zynqmp-zcu102-rev1.0.dts
+++ b/arch/arm64/boot/dts/xilinx/zynqmp-zcu102-rev1.0.dts
@@ -34,7 +34,3 @@
 		reg = <0xe0 0x3>;
 	};
 };
-
-&sdhci1 {
-	/delete-property/ no-1-8-v;
-};
-- 
2.7.4

